book/website/community-handbook/local-build.md

```diff
@@ -1,154 +1,184 @@
 (ch-local-build)=
-# Build the Turing Way Book Locally
+# Build the Book Locally
 
-## But why build locally?
+## Why build locally
 
-It's always handy to be able to preview any changes you have been working on as you go - you can be confident that changes you have made are accurate and as intended.
-A nice way to do this is to use the underlying [Jupyter Book](https://jupyterbook.org/en/stable/intro.html) tool to build the book locally.
+It is useful to preview changes you have been working on as you go on your local machine.
+You can be confident that changes you have made are accurate and as intended and it will likely be quicker than waiting for a preview to be build from a pull request.
+You can replicate the build process using [Make](rr-make) and [Jupyter Book](https://jupyterbook.org/en/stable/intro.html).
 
-This is useful because it allows you to preview any changes you have made on your local machine *before* you push your changes to a remote branch.
-You can then decide if you are happy with the result and push your changes to the remote branch thus helping to keep Pull Request conversations and commit histories a bit cleaner.
-
-## Step-by-step guide
+## Prerequisites
 
 We will be using the command line throughout this guide.
-You will need to locate your "terminal" or "prompt" application on your machine.
+You will need to use a terminal emulator to follow.
 
-1. Install miniconda by following the instructions for your Operating System (Windows, MacOS, Linux) at this link: https://conda.io/projects/conda/en/latest/user-guide/install/index.html#regular-installation
+You will also need to install Python3.
+You can check which specific version of Python3 the build uses in [netlify.toml](https://github.com/the-turing-way/the-turing-way/blob/main/netlify.toml).
+However, versions close to that version are also likely to work.
```

Comment:
I think it would have helped if the infrastructure team could also review this. What does a terminal emulator mean here?

Also, what is python3? that's the only cpython we have, and it's just a side effect of a couple of systems that they keep calling it python3.  And the netlify config doesn't use it, so even just refer to the env variable here would make more sense.

Reply:
I'm happy for suggestions/changes if we can improve the document :rocket:.

I thought terminal emulator was a common term, but if it doesn't make sense to you I think I must have been wrong :laughing:. What would be a better term, terminal, console, the command line, shell?

Was in two minds about specifying Python3. I'm sure it is safer now than before, but I think a number of package managers still have `python` as Python2 or null. Ubuntu even has a `python-is-python3` package that just symlinks `python` to `python3`.

> And the netlify config doesn't use it, so even just refer to the env variable here would make more sense.

Not sure what this means.